cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,33 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+// UpdateConfigHandler returns handler for runtime configuration changes.
+func UpdateConfigHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		_ = req.ParseForm()
```

Comment:
```suggestion
		if err := req.ParseForm(); err != nil {
		    httpError(http.StatusBadRequest, err)
		    return
		}
```
Errors here are not acceptable